far/config.cpp

```diff
@@ -489,6 +489,7 @@ void Options::MaskGroupsSettings()
 	const auto BottomTitle = KeysToLocalizedText(KEY_INS, KEY_DEL, KEY_F4, KEY_F7, KEY_CTRLR);
 	MasksMenu->SetBottomTitle(BottomTitle);
 	MasksMenu->SetHelp(L"MaskGroupsSettings"sv);
+        MasksMenu->SetId(MaskGroupsMenuId);
```

Comment:
Fix the formatting please: spaces -> tabs.
Same in the other file.

Reply:
Done